assets/src/dashboard/components/popover-menu/index.js

```diff
@@ -140,20 +146,27 @@ const PopoverMenu = ({
     setHoveredIndex(0);
   }, [items]);
 
-  const renderMenuItem = (item, index) => {
-    const itemIsDisabled = !item.value && item.value !== 0;
-    return (
-      <MenuItem
-        key={`${item.value}_${index}`}
-        isHovering={index === hoveredIndex}
-        onClick={() => !itemIsDisabled && onSelect && onSelect(item)}
-        onMouseEnter={() => setHoveredIndex(index)}
-        isDisabled={itemIsDisabled}
-      >
-        <MenuItemContent>{item.label}</MenuItemContent>
-      </MenuItem>
-    );
-  };
+  const renderMenuItem = useCallback(
+    (item, index) => {
+      const itemIsDisabled = !item.value && item.value !== 0;
+      return (
+        <MenuItem
```

Comment:
would an alternative to the li/span break be checking for a separator and passing it as a prop to the menuItem and rendering a border?